spanner/session.go

```diff
@@ -694,6 +749,18 @@ func newSessionPool(sc *sessionClient, config SessionPoolConfig) (*sessionPool,
 			return nil, err
 		}
 	}
+	if pool.enableMultiplexSession {
+		go pool.createMultiplexedSession()
+		pool.multiplexedSessionReq <- muxSessionCreateRequest{force: true, ctx: context.Background()}
+		// listen for the session to be created
+		go func() {
+			select {
+			// wait for the session to be created
+			case <-pool.mayGetMultiplexedSession:
+			}
+			return
+		}()
```

Comment:
The only reason we are doing this is because sessionConsumer upon failure/success does 
```
p.mayGetMultiplexedSession <- true
```
But if there are no routines listening on mayGetMultiplexedSession the above operation will be stuck.